modules/openxOrtbBidAdapter.js

```diff
@@ -47,182 +138,22 @@ function isBidRequestValid(bidRequest) {
 function buildRequests(bids, bidderRequest) {
   let videoBids = bids.filter(bid => isVideoBid(bid));
   let bannerBids = bids.filter(bid => isBannerBid(bid));
-  let requests = bannerBids.length ? [createBannerRequest(bannerBids, bidderRequest)] : [];
+  let requests = bannerBids.length ? [createRequest(bannerBids, bidderRequest, BANNER)] : [];
   videoBids.forEach(bid => {
-    requests.push(createVideoRequest(bid, bidderRequest));
+    // TODO: why is this creating a request for each bid, instead of aggregating them together as with banner?
```

Comment:
this is a limitation in our exchange currently. only banner requests can be aggregated, video requests only support a single slot